server/conf/routes

```diff
@@ -100,6 +100,9 @@ GET     /admin/apiKeys/new                          controllers.admin.AdminApiKe
 POST    /admin/apiKeys                              controllers.admin.AdminApiKeysController.create(request: Request)
 POST    /admin/apiKeys/:apiKeyId/retire             controllers.admin.AdminApiKeysController.retire(request: Request, apiKeyId: Long)
 
+GET     /admin/feature/:flag/enable                 controllers.FeatureFlagOverrideController.enable(request: Request, flag: String)
```

Comment:
Since this is only intended for use in non-production (i.e. dev) environments, what about grouping them with the existing `/dev/` routes (e.g. `/dev/features/enable`?

Reply:
done